app/Http/Controllers/User/MyRequestController.php

```diff
@@ -81,6 +84,20 @@ public function update(Request $request, $id)
             $this->notification->store($data);
         }
 
+        if ($request->status == 'returning') {
+            $book_id = $this->bookUser->findId($id)->book_id;
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/User/MyRequestController.php</br>**Line**: 88</br>Variable "book_id" is not in valid camel caps format